src/spi.rs

```diff
@@ -41,6 +41,7 @@ use crate::pac::{SPI1, SPI2};
 
 use crate::afio::MAPR;
 use crate::dma::dma1::{C3, C5};
+use crate::dma::dma2::C2;
```

Comment:
Could you also `cfg` this to avoid the warnings when compiling for other devices ?